modules/phase_field/tutorials/spinodal_decomposition/s4_mobility.i

```diff
@@ -0,0 +1,208 @@
+#
+# Example simulation of an iron-chromium alloy at 500 C. Equilibrium
+# concentrations are at 23.6 and 82.3 mol% Cr. Kappa value, free energy equation,
+# and mobility equation were provided by Lars Hoglund. Solved using the split
+# form of the Cahn-Hilliard equation.
+#
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  distribution = DEFAULT
+  elem_type = QUAD4
+  nx = 25
+  ny = 25
+  nz = 0
+  xmin = 0
+  xmax = 25
+  ymin = 0
+  ymax = 25
+  zmin = 0
+  zmax = 0
+  uniform_refine = 2
+[]
+
+[GlobalParams]
+  block = 0           # The generated mesh is used for all materials and kernels
+  enable_jit = true   # Enables Just-in-time compilation for parsed materials
```

Comment:
I think that just-in-time compilation is enabled by default, so it is actually being run in every one of these input files